src/templates/EditorialNewsletter/schema.js

```diff
@@ -72,39 +72,41 @@ const createNewsletterSchema = ({
     editorModule: 'link'
   }
 
-  const paragraph = {
-    matchMdast: matchParagraph,
-    component: Paragraph,
-    editorModule: 'paragraph',
-    editorOptions: {
-      formatButtonText: 'Paragraph'
-    },
-    rules: [
-      ...globalInlines,
-      link,
-      {
-        matchMdast: matchType('strong'),
-        component: ({ attributes, children }) => (
-          <strong {...attributes}>{children}</strong>
-        ),
-        editorModule: 'mark',
-        editorOptions: {
-          type: 'STRONG',
-          mdastType: 'strong'
-        }
+  const paragraph = customComponent => {
```

Comment:
```suggestion
  const createParagraphRule = customComponent => {
```

and then just call it twice:
```js
const paragraph = createParagraphRule()
const listParagraph = createParagraphRule(ListP)
```